src/d_loop.c

```diff
@@ -730,7 +730,7 @@ void TryRunTics (void)
     // [AM] If we've uncapped the framerate and there are no tics
     //      to run, return early instead of waiting around.
     // Only check counts so that I_FinishUpdate() can handle frame timing.
```

Comment:
I added this line previously but it's best to remove it now since it's no longer accurate: `// Only check counts...`

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1492/commits/1f72366e616dfd6ad54b118280df5d55e002a009